src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -800,21 +760,53 @@ TEST_CASE("Request rate - Check setting of InitSharedMemory")
   bool is_sequence = false;
   bool is_decoupled = false;
   bool use_mock_infer = true;
+  const std::string json_str{R"(
```

Comment:
Shouldn't be needed, but I think those whole test can go away anyways

Reply:
Removed